data/collections/bangladesh-landcover-2001-2020.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "bangladesh-landcover-2001-2020",
+    "type": "Collection",
+    "links": [],
+    "title": "Annual land cover maps for 2001 and 2020",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    88.02591469087191,
+                    20.742099910319755,
+                    92.68367943903164,
+                    26.63504817414382
+                  ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2001-01-01T00:00:00Z",
+                    "2020-12-31T23:59:59Z"
+                ]
+            ]
+        }
+    },
+    "license": "CC0",
```

Comment:
Tiny change: If you change this id to CC0-1.0 community tools like the browser will link out to the SPDX record https://spdx.org/licenses/CC0-1.0.html

Reply:
Thank you! Done.